### DISCUSSION TITLE
Default voice assistant

### DISCUSSION THREAD

**Discussion by Pseudoruse** [BimmerGestalt/AAIdrive#222] [Q&A]
The app works extremely well, it was absolutely painless to get up and running. 

The only thing I cannot seem to change is what happens when I hold the voice assistance button on my steering wheel. As I have a Samsung S20, it defaults to Bixby which I have never set up or intend on using. Google assistant is also there, however is second on the list in the app, under Bixby; I'm not sure if the order of it matters. 

I do have google as the default voice assistant within my phones settings, however the AndroidAutoIdrive app seems to prefer Bixby. I've tried different methods to change which one is used. Perhaps I am missing something obvious. Anyone have any suggestions on how to set Google as default?

**Comment by hufman** [comment]
Thank you for the kind words! I'm glad it's working well for you!
The assistants discovered and added by Android Auto IDrive exist only in the Online Services menu, and are separate from the steering wheel assistant button. I added this feature because some cars (I think IDrive4 and all Minis) don't have steering wheel functionality to trigger the assistant. Your car, with the steering wheel button, triggers the system-wide default assistant.

It looks like you can follow [these instructions](https://www.samsung.com/us/support/answer/ANS00077672/) on your Samsung phone to change the system default.

**Reply by Pseudoruse** [reply]
I see. As I said in the above post, google is the default.

Maybe I just need to mess with it some more in the car.

**Reply by bogdan-calapod** [reply]
Try resetting defaults (e.g. having no voice assistant as default) and then long press the steering wheel button while watching the phone screen. It should ask you which assistant you want to use :D 

@hufman maybe some kind of ℹ button in the app somewhere with instructions on this would help ? :D

**Reply by jagerx9** [reply]
It would be easier if you just disabled Bixby on your s20 as it sounds like you don't want to use it. Install "minimal fastboot and ADP" then look up instructions to remove Samsung bloatware

**Reply by kalahcustom** [reply]
> Try resetting defaults (e.g. having no voice assistant as default) and then long press the steering wheel button while watching the phone screen. It should ask you which assistant you want to use :D
> 
> @hufman maybe some kind of ℹ button in the app somewhere with instructions on this would help ? :D

Confirmed. Works fine for me too (iDrive 6). 👍
Settings -> Applications -> Bixby Voice -> Setting as default -> Clear default settings
